src/main/java/rx/BackpressureOverflow.java

```diff
@@ -24,6 +24,30 @@
 @Experimental
 public final class BackpressureOverflow {
 
+    /**
+     * By default, signal a MissingBackressureException due to lack of requests.
+     */
+    public static final BackpressureOverflow.Strategy ON_OVERFLOW_DEFAULT = Error.INSTANCE;
```

Comment:
Would be nice to move this below all other strategies and change assignment to `ON_OVERFLOW_DEFAULT = ON_OVERFLOW_ERROR`  so it'll be more clear that default strategy is same as `ON_OVERFLOW_ERROR`